commerce_coordinator/apps/frontend_app_ecommerce/tests/test_filters.py

```diff
@@ -0,0 +1,31 @@
+"""
+Tests for the frontend_app_ecommerce app filters.
+"""
+
+from django.test import TestCase
+from mock import patch
+
+from commerce_coordinator.apps.frontend_app_ecommerce.filters import OrderHistoryRequested
+from commerce_coordinator.apps.frontend_app_ecommerce.tests import (
+    ECOMMERCE_REQUEST_EXPECTED_RESPONSE,
+    ORDER_HISTORY_GET_PARAMETERS
+)
+from commerce_coordinator.apps.frontend_app_ecommerce.tests.test_views import EcommerceClientMock
+
+
+@patch('commerce_coordinator.apps.ecommerce.clients.EcommerceApiClient.get_orders',
+       new_callable=EcommerceClientMock)
+class TestFrontendAppEcommerceFilters(TestCase):
+    """
+    Test class to verify standard behavior of the frontend_app_ecommerce filters.
+    """
+    # Disable unused-argument due to global @patch
+    # pylint: disable=unused-argument
+
+    def test_order_history_requested_filter(self, mock_ecommerce_client):
+        """
+        Confirm that OrderHistoryRequested filter calls expected PipelineStep: GetEcommerceOrders
+        """
+
+        response = OrderHistoryRequested.run_filter(ORDER_HISTORY_GET_PARAMETERS)
+        self.assertEqual(response, ECOMMERCE_REQUEST_EXPECTED_RESPONSE)
```

Comment:
Since we already have tests for individual steps along the way, right now this filter test just checks that the GetEcommerceOrders pipeline step gets called when we run the filter. (And it's roundabout: if we patch EcommerceApiClient and that's what comes back, then the code tried to go there). Come back and look at what we may want to add.

Reply:
Maybe add `@override_settings` for `OPEN_EDX_FILTERS_CONFIG`? We don't have to use `@override_settings` now because `OPEN_EDX_FILTERS_CONFIG` is already wired for this test. However, as we make changes to the order history filter pipeline configuration or shift it into our internal config repos, this test may break.

What I hope this test can do for us:
1. Check the `OrderHistoryRequested` filter calls `EcommerceApiClient.get_orders` when `GetEcommerceOrders` is in the pipeline
2. Check upgrades to openedx_filters's `run_filter` don't break our pipelines
3. Show in `@override_settings` how to configure `OPEN_EDX_FILTERS_CONFIG` to get the test's behavior
4. Be a template for future, perhaps wider tests (e.g. with multiple filters)

Reply:
Thanks Phil! Interesting idea to override what's in OPEN_EDX_FILTERS_CONFIG. My intent here was actually to make sure that ecommerce isn't accidentally removed from the pipeline along the way (in which case I definitely don't want to override those settings), but I see that my test name is vague and only the comment is explicit. 

Do you think it's worth two tests: 
- keep the current one as-is, as a fail safe that we don't accidentally take ecommerce out of the pipeline (or when we do intentionally, we remove this test)
- add a more general test whose purpose is to make sure nothing has broken with openedx_filters's `run_filter` or with our pipeline configuration in general

Or is adding a second test overkill since those issues would be caught by the test that's there?

Reply:
As-is is good. We may need to do `@override_settings` (the second test) once we deprecate ecommerce, but that can be a problem for the future.